CHANGELOG.md

```diff
@@ -31,6 +31,9 @@
 * Generate JS bindings for WebIDL dictionary setters instead of using `Reflect`. This increases the size of the Web API bindings but should be more performant. Also, importing getters/setters from JS now supports specifying the JS attribute name as a string, e.g. `#[wasm_bindgen(method, setter = "x-cdm-codecs")]`.
   [#3898](https://github.com/rustwasm/wasm-bindgen/pull/3898)
 
+* Greatly improve the performance of sending WebIDL 'import enums' across the JavaScript boundary by converting the enum variant string to/from an int.
```

Comment:
This should probably point out that this affects any enum definitions, not just WebIDL.

Reply:
Ive been calling them ImportEnums but they should really be called StringEnums as per the docs: https://rustwasm.github.io/wasm-bindgen/reference/types/imported-js-types.html. I think that's a better name. 

Shall I rename them across the board in the code? "ImportEnum" is a misnomer.